### PR TITLE
Add notification on authenticator verification

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

Fixes https://github.com/appwrite/console/issues/1298

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

* https://github.com/appwrite/console/issues/1298

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/console/account/mfa.svelte [hunk 1]
```diff
@@ -28,6 +29,10 @@
             await sdk.forConsole.account.updateMfaAuthenticator(AuthenticatorType.Totp, code);
             await Promise.all([invalidate(Dependencies.ACCOUNT), invalidate(Dependencies.FACTORS)]);
             showSetup = false;
+            addNotification({
+                type: 'success',
+                message: 'Authenticator app added successfully'
```
Comment (ItzNotABug) [inline src/routes/console/account/mfa.svelte:34]: not sure if we should use `added` or `connected` here.
The status `Pill` component below uses `connected` though.

Comment (ItzNotABug) [reply src/routes/console/account/mfa.svelte]: @sara-k-48 done.

**Review by christyjacob4 — APPROVED** [review]
Looks good code wise. Please make sure we get a design review done.

**Comment by ItzNotABug** [comment]
> Looks good code wise. Please make sure we get a design review done.

This has been approved for the time being.

**Comment by stnguyen90** [comment]
Merging because Christy approved earlier and Sara approved internally.

**Pull request merged**